.github/workflows/pr.yml

```diff
@@ -76,12 +74,13 @@ jobs:
       - name: Get auth token
         id: get-token
         run: |
+          echo "Vars: $vars"
           response=$(curl -X POST \
             ${{ vars.STAGING_COGNITO_DOMAIN }}/oauth2/token \
             -H "Content-Type: application/x-www-form-urlencoded" \
             -d "grant_type=client_credentials" \
             -d "client_id=${{ vars.STAGING_CLIENT_ID }}" \
-            -d "client_secret=${{ secrets.STAGING_CLIENT_SECRET }}"
+            -d "client_secret=${{ vars.STAGING_CLIENT_SECRET }}"
```

Comment:
Does this need to be stored in the environment secret?

Reply:
I think using a secret in this case would be appropriate.

Reply:
https://github.com/NASA-IMPACT/veda-data/pull/181/commits/e40df511961a99a3d58714c7c96bb01dd2a9d7bd